logexp/Makefile

```diff
@@ -0,0 +1,28 @@
+# Copyright 2017 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+IMG = gcr.io/google-containers/logexp
+TAG = v0.1.0
+
+all: build
+
+build: logexp.sh
+	docker build -t $(IMG):$(TAG) .
+	docker tag $(IMG):$(TAG) $(IMG):latest
```

Comment:
This is a convention I followed from https://github.com/kubernetes/test-infra/blob/master/jenkins/e2e-image/Makefile#L26
And usually it is suggested to not have prod containers run against latest tag. For a volatile component in test, especially like this one, which is tagged with a computed hash and timestamp, the latest tag might be convenient. WDYT?